content/en/docs/refguide/modeling/application-logic/workflows/workflow-elements/boundary-events/timer-event.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Timer Event"
+url: /refguide/timer-event/
+weight: 10
+---
+
+{{% alert color="info" %}}
+This feature is currently in public beta as of version 10.15. Please note that certain features or attributes may be subject to change and may contain bugs.{{% /alert %}}
+
+## Introduction
+
+The **Timer Event** enables the timed-execution of alternative flows when the guarded activity is exceeding the set timer.
+
+## Properties
+
+**Timer Event** properties consist of the following sections:
+
+* [General](#general)
+* [Timer](#timer)
+* [Common](#common)
+
+### General Section {#general}
+
+The **Caption** describes what happens in this element. It is displayed on the workflow element to make the **Timer** event easier to read and understand.
+
+### Timer Section {#timer}
+
+The **Timer** property specifies the exact moment or time interval after which the workflow progresses to the next step. With this property, you can set the timer in two ways: 
+
+1. you can set a certain duration
+2. certain date and time with an expression
+
+When an activity finishes before the **Timer** has elapsed, and the workflow path reaches the timer, the configured duration or date and time will be scheduled to take effect.
+
+The **Timer** properties are described in the table below:
+
+| Type | Description |
+| --- | --- |
+| Duration | You can set a certain duration for the timer. With the **Continue after** setting, you can indicate the number of seconds, minutes, hours, days, weeks or months the timer's duration is. Possible values for the setting are:<br /><ul><li>Seconds</li><li>Minutes</li><li>Hours</li><li>Days</li><li>Weeks</li><li>Months</li> </ul> |
+| Expression | You can set a certain date and time for the timer by writing an expression via the **Continue at** setting.<br><br>For example, you can write `addDays([%CurrentDateTime%], 1)` to set tomorrow as the due date and time. To set a static date and time, you can use the expression `parseDateTimeUTC('2023-12-10T17:12:00.000', 'yyyy-MM-dd''T''HH:mm:ss.SSS')`.<br><br>You can also create a more complex timer. For example, you can set a timer based on a Boolean value (in this example, `isVIPUser`) from the provided workflow context entity: `if $WorkflowContext/isVIPUser then addDays([%CurrentDateTime%], 2) else addWeeks([%CurrentDateTime%], 2])`.<br><br>For more information on available expressions in Mendix, see [Expressions](/refguide/expressions/). |
+
+## Timer Expiration {#timer-expiration}
+
+When a **Timer** event expires, it behaves differently depending on the state of the workflow:
+
+* * When a timer event is set on an in-progress workflow, the workflow continues when the timer expires, and will continue the flow.
+
+* When a timer event is set on a paused workflow and when the timer expires, the workflow only starts the flow after the workflow is set to in progress again.
```

Comment:
This item refers to the behavior from `Specific Workflow State Cases`, where it is described in more detail. I think we should combine the sections, or move the items from  `Specific Workflow State Cases` to the `Timer Expiration` section